ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -127,8 +129,8 @@ function populate_artifact_dir() {
   # Copy CAPI-generated artifacts if they exist
   if [ -d "${dir}/.clusterapi_output" ]; then
     echo "Copying Cluster API generated manifests..."
-    mkdir -p "${ARTIFACT_DIR}/clusterapi_output/"
-    cp -rpv "${dir}/.clusterapi_output/"{,**/}*.{log,yaml} "${ARTIFACT_DIR}/clusterapi_output-$(date +%s)" 2>/dev/null
+    mkdir -p "${ARTIFACT_DIR}/clusterapi_output-${current_time}"
+    cp -rpv "${dir}/.clusterapi_output/"{,**/}*.{log,yaml} "${ARTIFACT_DIR}/clusterapi_output-${current_time}" 2>/dev/null
```

Comment:
From the above change, I can see now we are using a consistent timestamp file name. Does it help resolve `saving content under <install_dir>/.clusterapi_output seems not working well` issue ?

Reply:
`ipi-install-install` has retry mechanism to create cluster if previous one is failed due to infrastructure problems, after changing to use consistent timestamp file name for openshift-install.log and terraform/clusterapi_output, we could easily know the logs with same timestamp are from same installation.